src/test/java/core/basesyntax/service/impl/ReadFromFileImplTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.exception.FruitShopException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReadFromFileImplTest {
+    private static final ReadFromFileImpl readFromFile = new ReadFromFileImpl();
+    private static final String path = "src/test/resources/Hello.csv";
+    private static final String expectedString = "Hello, mate!";
+
+    @BeforeClass
+    public static void beforeClass() {
+        try {
+            Files.writeString(Paths.get(path), expectedString);
+        } catch (IOException e) {
+            throw new RuntimeException("Check the file path " + path, e);
+        }
+    }
+
+    @Test
+    public void readFile_isContentEqual_Ok() {
+        String actualString;
+        try {
+            actualString = Files.readString(Paths.get(path));
+        } catch (IOException e) {
+            throw new FruitShopException("Enter correct file path");
+        }
+        boolean isContentEqual = actualString.equals(expectedString);
+        assertTrue(isContentEqual);

```

Comment:
I think better create two String  variables first with method readString second with expected and equal them with assertEquals

Reply:
fixed